backend/src/common/runner/index.ts

```diff
@@ -159,6 +169,11 @@ const create_images = () => {
     );
     if(ret?.stdout)
         console.log(ret.stdout.toString());
+    console.log(`Image ${img}/${img_ver} created`);
+};
+const createImages = () => {
+    console.log("creating images");

```

Comment:
did u forget to remove these

Reply:
or at least maybe we can separate logging into error, info, warn, etc..?

Reply:
we use log for all backend logging

Reply:
Fixed on the entire backend now